modules/gitops-release-notes-1-11-3.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assembly:
+//
+// * release_notes/gitops-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+
+[id="release-notes-for-gitops-1-11-3_{context}"]
+= Release Notes for {gitops-title} 1.11.3
+
+{gitops-title} 1.11.3 is now available on {OCP} 4.12, 4.13, and 4.14.
+
+[id="errata-updates-1-11-3_{context}"]
+== Errata updates
+
+[id="rhsa-2024-1697-gitops-1-11-3-security-update-advisory_{context}"]
+=== RHSA-2024-1697 - {gitops-title} 1.11.3 security update advisory
```

Comment:
Incorrect format for an advisory:
```suggestion
=== RHSA-2024:1697 - {gitops-title} 1.11.3 security update advisory
```
Out of scope, but there are a handful of other previous sections in this assembly where it is using the `-` style instead of the correct `:`. Suggest opening a follow-up PR to clean them all up at once. It would also help for future PRs that might be starting from these older modules to duplicate a new section.

Reply:
Fixed.